uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Hook/PageRun/RelationshipTypesFilter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+class CRM_HRCore_Hook_PageRun_RelationshipTypesFilter {
+
+  /**
+   * Hides disabled relationship types in the interface for non-root admins
+   *
+   * @param CRM_Core_Page $page
+   */
+  public function handle($page) {
+    if (!$this->shouldHandle($page)) {
+      return;
+    }
+
+    $rows = $page->get_template_vars('rows');
+    // remove disabled relationship types
+    foreach ($rows as $index => $row) {
+      if (CRM_Utils_Array::value('is_active', $row) == '0'&& in_array(CRM_Utils_Array::value('name', $row),['Case Coordinator is','Employee of','Head of Household for','Household member of'])) {
```

Comment:
Did you run `civilint` on this file? I see a missing space after `'0'`. Also this line is very long, please extract the variables to make it more readable like 

`['Case Coordinator is','Employee of','Head of Household for','Household member of']`  => `$targetRelTypes`
`CRM_Utils_Array::value('is_active', $row) == '1'` => `$isActive`
`CRM_Utils_Array::value('name', $row)` => `$relationshipName`
`in_array($relationshipName, $targetRelTypes)` => `$isTargetType`

So you'll end up with a more readable conditional like

`if (!$isActive && $isTargetType)`

The variable names are just suggestions, change them to whatever you like